tests/test_evaluate.py

```
@@ -681,3 +699,724 @@
+                frame_idx=1,
+                model_version="002",
+            )
+
+        # Verify function calls - just check that it runs without error
+        mock_wandb.init.assert_called_once()
+        mock_fetch_artifact.assert_called_once()  # Called once per group, we have 1 group
+
+    @patch("sleap_roots_training.evaluate.predictions_viz")
+    def test_predictions_viz_multiple_files_coverage(self, mock_predictions_viz):
+        """Test predictions_viz_multiple_files for coverage."""
+        model_artifacts = ["model1", "model2"]
```

Ruff (0.12.2)

Local variable `model_artifacts` is assigned to but never used

Remove assignment to unused variable `model_artifacts`

(F841)